SUBGROUPS.md

```diff
@@ -84,20 +84,21 @@
 * Status
   * Active
 
-## Events and Outreach
+## Outreach Committee
 
 * Description & Responsibilities
-  * The charter of the subgroup is to plan, coordinate, and execute DB
-    community and other industry events as well as lead outreach, marketing,
-    and PR activities for DB
+  * The charter of the subgroup is where recruiting for additional project members,
+  events coordination, vision discussions, and marketing related activities will

```

Comment:
Can you move these sentences to line up with `The` above? I'm somewhat surprised this passed the linter.

Reply:
done